e2e/playwright/support/server/default_config.ts

```diff
@@ -259,7 +259,7 @@ const defaultServerConfig: AdminConfig = {
         EnableMobileUpload: true,
         EnableMobileDownload: true,
         MaxFileSize: 104857600,
-        MaxVoiceMessagesFileSize: 104857600,
+        MaxVoiceMessagesDuration: 0,
```

Comment:
Should we set a sensible default to avoid potential abuse? Like 300 (5 minutes) or something like that.

Reply:
changed!